plop-templates/@react-spectrum/stories/Component.stories.tsx.hbs

```diff
@@ -11,23 +11,29 @@
  */
 
 import {action} from '@storybook/addon-actions';
-{{#if (includes scopes "@react-types")}}
 import { {{~componentName~}} } from '../';
+import {Meta, Story} from '@storybook/react';
+import React from 'react';
+{{#if (includes scopes "@react-types")}}
 import {Spectrum{{componentName}}Props} from '@react-types/{{packageName}}';
-{{else}}
-import { {{~componentName}}, Spectrum{{componentName}}Props} from '../';
 {{/if}}
-import React from 'react';
-import {storiesOf} from '@storybook/react';
+{{#unless (includes scopes "@react-types")}}
 
-storiesOf('{{componentName}}', module)
-  .add(
-    'name me',
-    () => render({})
-  );
+interface Spectrum{{componentName}}Props {
 
-function render(props: Spectrum{{componentName}}Props) {
-  return (
-    <{{componentName}} {...props} onChange={action('onChange')} />
-  );
 }
+{{/unless}}
+
+const meta: Meta<Spectrum{{componentName}}Props> = {
+  title: '{{componentName}}',
+  component: {{componentName}}
+};
+
+export default meta;
+
+const Template = (): Story<Spectrum{{componentName}}Props> => (args) => (
+  <{{componentName}} {...args} onChange={action('onChange')}>This is a React Spectrum {{componentName}}</{{componentName}}>
```

Comment:
I think it'd be better to move the onChange into the Default's args, this way it can be omitted from a story if needed

Reply:
Yup, done 👍🏼